pkg/bkctl/ledger/read_test.go

```diff
@@ -23,6 +23,15 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestReadCmd(t *testing.T) {
+	args := []string{"read", "1"}
+	_, execErr, nameErr, err := testLedgerCommands(readCmd, args)
+	assert.Nil(t, err)
+	assert.Nil(t, nameErr)
+	assert.NotNil(t, execErr)
+	assert.Equal(t, "code: 500 reason: Unknown pulsar error", execErr.Error())
```

Comment:
why this is an unknown pulsar error?

Reply:
Ah. I think that because the error message is different between pulsar and bookkeeper.  I think I need to refactor the error handler. I will do that with a separate pull request.

Reply:
Please create an issue for "I will do that with a separate pull request". People usually forget sending a pull request for that.